Auth0/SafariWebAuth.swift

```diff
@@ -153,6 +158,7 @@ class SafariWebAuth: WebAuth {
             entries["nonce"] = self.nonce
         }
         self.parameters.forEach { entries[$0] = $1 }
+        self.parameters = [:]
```

Comment:
In a WebAuth instance I noticed if you cancel a connection then try another connection the parameters will persist.